testsuite/forge/src/backend/k8s/node.rs

```diff
@@ -161,17 +166,28 @@ impl Node for K8sNode {
     }
 
     fn clear_storage(&mut self) -> Result<()> {
-        let sts_name = self.stateful_set_name.clone();
-        let pvc_name = if sts_name.contains("fullnode") {
-            format!("fn-{}-0", sts_name)
-        } else {
-            sts_name
-        };
-        let delete_pvc_args = ["delete", "pvc", &pvc_name];
-        info!("{:?}", delete_pvc_args);
+        let ledger_db_path = format!("{}/db/{}", APTOS_DATA_DIR, LEDGER_DB_NAME);
```

Comment:
In the future we should consider maybe extracting this functionality across both the local and k8 swarms (so we don't need to enumerate all the files twice). But this is good enough for now! :D